src/main/scala/org/apache/commons/configuration/SubsetConfigurationMethods.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.configuration
+
+object SubsetConfigurationMethods {
+
+  implicit class SubsetConfigurationOps(val conf: SubsetConfiguration) extends AnyVal {
+
+    // The `SubsetConfiguration.getParentKey()` method is protected,
+    // so we apply this implicit proxy to make the method publicly
+    // accessible without resorting to the reflection.
+    def getParentKey: String => String = conf.getParentKey
```

Comment:
Ok, but since this is "only" for a nice error message, using the public `getPrefix` and some hard-coded delimiter might have been good enough.

Reply:
Technically yes, but why would I hardcode a piece (even a tiny one) of a 3rd party library class logic when I can call a dedicated method? I also don't want to handle empty prefix cases.